dpnp/backend/examples/example_bs_sycl.cpp

```diff
@@ -0,0 +1,97 @@
+#include <CL/sycl.hpp>
+#include <iostream>
+
+cl::sycl::event divide_sycl(double* price,
+                            double* strike,
+                            const size_t size,
+                            double* p_div_s,
+                            cl::sycl::queue q)
```

Comment:
I would propose to order parameters of this and below functions as in MKL, e.g. https://spec.oneapi.com/versions/latest/elements/oneMKL/source/domains/vm/div.html.
```suggestion
cl::sycl::event divide_sycl(cl::sycl::queue q,
                            const size_t size,
                            double* price,
                            double* strike,
                            double* p_div_s)
```
Other solution is following DPNP order.